automates/text_reading/src/test/scala/org/clulab/aske/automates/text/TestFunctions.scala

```diff
@@ -314,9 +365,92 @@ class TestFunctions extends ExtractionTest {
     testFunctionEvent(mentions, desired)
   }
 
+  // Tests from 1985-description and performance of CERES-Wheat_A_user-oriented wheat yield model
+
+  val t1i = "The primary variable influencing development rate is temperature."
+  failingTest should s"find functions from t1i: ${t1i}" taggedAs(Somebody) in {
+    val desired = Seq(
+      "development rate" -> Seq("temperature") //note: overlap between trigger and output??
```

Comment:
The problem here might be that the rule requires the output arg be a Phrase and development rate is not a phrase, but part of a phrase... In this case I frequently end up with token rules, which would be a shame for this case because the syntax is there...

Reply:
Right... It would be great if we could have a various level(?) of phrases (for example, here, we'd like to have both "The primary variable influencing development rate" and "development rate" as phrases) and allow rules to be applied to different level of phrases when needed. I'm not sure if this idea is viable, though.